.github/workflows/release.yml

```diff
@@ -74,7 +74,7 @@ jobs:
         with:
           tag_name: ${{ needs.tag.outputs.tag }}
           body: ${{ steps.changelog_reader.outputs.changes }}
-          prerelease: true
+          prerelease: false
```

Comment:
@mpetrunic I am not just sure if I should do this when merging the next release on master or should I change it here?

Reply:
here

Reply:
@mpetrunic ^^

Reply:
![image](https://user-images.githubusercontent.com/8836210/106136970-5840c700-616a-11eb-8227-b056a64521a5.png)